graph/min_cost_b_flow/gen/anti_ssp_00.in

```diff
@@ -0,0 +1,704 @@
+52 651
```

Comment:
I guess you generated this case by some program, not hand. So could you add this case by ".cpp", not ".txt"? Though the size of this case is okay to put directly, I prefer ".cpp" because we can view the feature of case easily.

Reply:
Sure! It also going to be useful if we ever want to change the constraints.